client/controller/index.node.js

```diff
@@ -30,3 +30,13 @@ const ReduxWrappedLoggedOutLayout = ( { store, primary, secondary } ) => (
  * `context.primary` and `context.secondary` to populate it.
 */
 export const makeLayout = makeLayoutMiddleware( ReduxWrappedLoggedOutLayout );
+
+export function redirectLoggedIn( { isLoggedIn, res }, next ) {
+	// TODO: Make it work also for development env
```

Comment:
As discussed on Slack with @ockham:

> That's a recurring issue with SSR development, since due to our cookie-based auth, your local dev server can't know you're logged in.
> The issue is limited to `development`, I think it's fine to use that condition`

It properly redirects on the client side if server side redirect fails.